corehq/apps/userreports/views.py

```diff
@@ -234,7 +234,7 @@ def choice_list_api(request, domain, report_id, filter_id):
 
     def get_choices(data_source, filter, search_term=None, limit=20):
         table = get_indicator_table(data_source)
-        sql_column = table.c[filter.name]
+        sql_column = table.c[filter.field]
```

Comment:
Here we tried to reference the column by the `filter.name`, which was the filter slug. If the slug was not the name of the column, this raised an exception. Reference it by `filter.field` instead.